drivers/include/periph/adc.h

```diff
@@ -30,6 +30,18 @@
  * waiting for the result of a conversion (e.g. through putting the calling
  * thread to sleep while waiting for the conversion results).
  *
+ * @section     sec_pm (Low-) power implications
```

Comment:
e.g., yes. (though I would prefer a more consistent prefixing `drivers_periph_adc_sec_pm` or `drivers_periph_adc_pm`). Then again: are you referring anywhere to this section? Do you have to label it manually?

```markdown
 * # (Low-)power implications
```

should suffice in that case.